standup-helper.sh

```diff
@@ -5,8 +5,8 @@ set -e
 # and run git-standup there directly from node (cwd did nothing), so
 # here we are.
 
-progname=$0
-standup=$(npm bin -g)"/git-standup"
+progname="$0"
```

Comment:
Sorry, I don't know much about Shell. What is that `"` good for?

Reply:
From what I can tell, it's doing nothing right now in the case of `progname`. Since the only time this variable is echoed out it is in a string already. IMO, these quotes should be un-committed and only added when there is a need for them.

Reply:
Although, the quotes added later in the script are guarding against say, someone putting a bash command in as an option that would then get executed when the script looks for the argument. It's just forcing string interpretation internally, no matter the contents provided.

Reply:
@Garbee @mojoaxel I'd be fine with removing them wherever they're not necessary. Let me know what you two decide :)